generator/sequence_transform_data.py

```diff
@@ -57,7 +57,7 @@
 KC_SEMICOLON = 0x33
 KC_1 = 0x1E
 MOD_LSFT = 0x0200
-KC_MAGIC_0 = 0x0100
+KC_SEQ_TOKEN_0 = 0x0100
```

Comment:
This is not a QMK keycode, so we shouldn't use KC to avoid confusion. It should use the same name we use for the define it generates IMO.

Reply:
Agreed. I was the one who insisted it not be called a keycode in the first place.

Reply:
ST instead of KC would be fine there imo